utilities/src/main/java/io/syndesis/qe/TestConfiguration.java

```diff
@@ -100,6 +102,8 @@ public static String openShiftUrl() {
 
     public static int getJenkinsDelay() { return Integer.parseInt(get().readValue(JENKINS_DELAY, "1")); }
 
+    public static String getDbAllocatorUrl(){ return get().readValue(DB_ALLOCATOR_URL, "http://dballocator.mw.lab.eng.bos.redhat.com:8080"); }
```

Comment:
The default value should be something not related to our infra. And the real value should be set in Jenkins configuration.